pyproject.toml

```diff
@@ -0,0 +1,12 @@
+[tool.ruff]
+# In addition to the standard set of exclusions, omit all tests, plus a specific file.
```

Comment:
I think test modules are still included and they should right?

Reply:
yes, correct!